sdk/clientcore/annotation-processor/pom.xml

```diff
@@ -110,7 +110,7 @@
     <dependency>
       <groupId>io.clientcore</groupId>
       <artifactId>core</artifactId>
-      <version>1.0.0-beta.7</version> <!-- {x-version-update;unreleased_io.clientcore:core;dependency} -->
+      <version>1.0.0-beta.7</version> <!-- {x-version-update;io.clientcore:core;dependency} -->
```

Comment:
We should also update this https://github.com/Azure/azure-sdk-for-java/blob/c5f3a736296bb5da798b34337e34e16486089b62/eng/versioning/version_client.txt#L499 and also remove if not needed.

Reply:
We still need as there are some other packages using the unreleased version.